pwa/src/RootComponents/CreateAccount/createAccount.js

```diff
@@ -0,0 +1,277 @@
+import React from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { Form } from 'informed';
+import { func, shape, string, bool } from 'prop-types';
+import { useCreateAccount } from '@magento/peregrine/lib/talons/CreateAccount/useCreateAccount';
+
+import { useStyle } from '@magento/peregrine/lib/context/style';
```

Comment:
```suggestion
import useStyle from '@magento/peregrine/lib/util/shallowMerge';
```
You can use the method from peregrine everywhere.